pkg/core/native/notary.go

```diff
@@ -52,7 +52,10 @@ const (
 	defaultMaxNotValidBeforeDelta = 140 // 20 rounds for 7 validators, a little more than half an hour
 )
 
-var maxNotValidBeforeDeltaKey = []byte{10}
+var (
+	maxNotValidBeforeDeltaKey = []byte{10}
+	activeIn                  = config.HFEchidna
```

Comment:
Don't forget it's package-level, better move into `ActiveIn()`.

Reply:
Sure, fixed.